specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -337,6 +337,22 @@ not required to upload.
 
   @doc("Configuration for quick load test")
   optionalLoadTestConfig?: OptionalLoadTestConfig;
+
+  @added(APIVersions.v2023_04_01_preview)
```

Comment:
Have you already shipped this API version? If so, you need to make a new version to add properties. Also make sure those generated swaggers are checked in.